src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -299,7 +306,43 @@ const ChatInfoScreen = () => {
           ListFooterComponent={
             <>
               {isLoadingFetchingChannelDetail ? <ActivityIndicator style={styles.loading} /> : null}
+              {channelInfo?.channelType === CHANNEL_GROUP && (
+                <View style={styles.btnAdd}>
+                  <TouchableOpacity
+                    testID="addParticipant"
+                    onPress={() =>
+                      navigation.push('ContactScreen', {
+                        from: SIGNED,
+                        isAddParticipant: true,
+                        channelId: channelInfo?.id,
+                        existParticipants: channelInfo?.memberUsers?.map((item) => item?.username)
+                      })
+                    }>
+                    <Text style={styles.btnAddText}>+ Add Participants</Text>
+                  </TouchableOpacity>
+                </View>
+              )}
               <View style={styles.gap} />
+              {channelInfo?.channelType === CHANNEL_GROUP && (
+                <View style={styles.actionGroup}>
+                  <TouchableOpacity onPress={onLeaveGroup} style={styles.buttonGroup}>
+                    <View style={styles.imageActContainer}>
+                      <FastImage style={styles.imageAction} source={ExitGroup} />
+                    </View>
+                    <View>
+                      <Text style={styles.textAct}>Exit Group</Text>
+                    </View>
+                  </TouchableOpacity>
+                  <TouchableOpacity onPress={onReportGroup} style={styles.buttonGroup}>
+                    <View style={styles.imageActContainer}>
+                      <FastImage style={styles.imageAction} source={ReportGroup} />
+                    </View>
+                    <View>
+                      <Text style={styles.textAct}>Report Group</Text>
+                    </View>
+                  </TouchableOpacity>
+                </View>
+              )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of "Add Participants", "Exit Group", and "Report Group" buttons is a good feature enhancement. It provides users with more control over the group chat. The usage of `dimen.normalizeDimen` function for setting dimensions is also a good practice as it helps to maintain consistency across different screen sizes and densities.
```diff
+               {channelInfo?.channelType === CHANNEL_GROUP && (
+                 <View style={styles.btnAdd}>
+                   <TouchableOpacity
+                     testID="addParticipant"
+                     onPress={() =>
+                       navigation.push('ContactScreen', {
+                         from: SIGNED,
+                         isAddParticipant: true,
+                         channelId: channelInfo?.id,
+                         existParticipants: channelInfo?.memberUsers?.map((item) => item?.username)
+                       })
+                     }>
+                     <Text style={styles.btnAddText}>+ Add Participants</Text>
+                   </TouchableOpacity>
+                 </View>
+               )}
+                <View style={styles.gap} />
+               {channelInfo?.channelType === CHANNEL_GROUP && (
+                 <View style={styles.actionGroup}>
+                   <TouchableOpacity onPress={onLeaveGroup} style={styles.buttonGroup}>
+                     <View style={styles.imageActContainer}>
+                       <FastImage style={styles.imageAction} source={ExitGroup} />
+                     </View>
+                     <View>
+                       <Text style={styles.textAct}>Exit Group</Text>
+                     </View>
+                   </TouchableOpacity>
+                   <TouchableOpacity onPress={onReportGroup} style={styles.buttonGroup}>
+                     <View style={styles.imageActContainer}>
+                       <FastImage style={styles.imageAction} source={ReportGroup} />
+                     </View>
+                     <View>
+                       <Text style={styles.textAct}>Report Group</Text>
+                     </View>
+                   </TouchableOpacity>
+                 </View>
+               )}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->